script/Deploy.s.sol

```diff
@@ -3,9 +3,14 @@ pragma solidity >=0.8.19 <=0.9.0;
 
 import { Foo } from "../src/Foo.sol";
 import { BaseScript } from "./Base.s.sol";
+import { DeploymentConfig } from "./DeploymentConfig.s.sol";
 
 contract Deploy is BaseScript {
-    function run() public broadcast returns (Foo foo) {
+    address private deployer;
```

Comment:
not sure how BaseScript works, is this used as the deployer in line 14?

Reply:
Ha, I totally mixed this up with tests.
The `deployer` should be made available in tests by exposing it from `run()`

Will fix this.